packages/kit/src/views/Send/components/CoinControlBadge/index.tsx

```
@@ -0,0 +1,44 @@
+import { ETranslations } from '@onekeyhq/shared/src/locale';
+
+type IProps = {
+  onPress?: () => void;
+};
+
+function CoinControlBadge({ onPress }: IProps) {
+  const intl = useIntl();
+  return (
+    <Badge
+      userSelect="none"
+      role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)